core/src/main/java/com/amplifyframework/auth/AuthUserAttribute.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.auth;
+
+import androidx.core.util.ObjectsCompat;
+
+public final class AuthUserAttribute {
```

Comment:
Yea, maybe. But, it looks a lot like an `androidx.core.util.Pair`. 

That might be so, but I do agree it is nice to own the entire surface of our own API, without relying on third-party types (AndroidX.)

You could, though, make `AuthUserAttribute` basically just a wrapper around a `Pair delegate` from `androidx.core.util.Pair`.

It's also so lightweight of a class that basically nothing I've said here matters lol :-D haha